### PR TITLE
PRLC100-148 Child Safety Concerns

### PR DESCRIPTION
**Pull request by ShakiraKP**
[### JIRA link (if applicable) ###](https://tools.hmcts.net/jira/browse/PRLC100-148)

C1A - Any other concerns about the Children's safety and wellbeing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/steps/respondent/respondentcaseSequence.test.ts [hunk 1]
```diff
@@ -118,17 +118,21 @@ describe('respondent1Sequence', () => {
...
-    expect(respondentCaseSequence[30].url).toBe('/respondent/proceedings/court-proceedings');
+    expect(respondentCaseSequence[30].url).toBe('/respondent/proceedings/start');
     expect(respondentCaseSequence[30].showInSection).toBe('aboutRespondentCase');
-    expect(respondentCaseSequence[30].getNextStep({})).toBe('/respondent/proceedings/summary');
 
-    expect(respondentCaseSequence[31].url).toBe('/respondent/proceedings/summary');
+    expect(respondentCaseSequence[31].url).toBe('/respondent/proceedings/court-proceedings');
     expect(respondentCaseSequence[31].showInSection).toBe('aboutRespondentCase');
-    expect(respondentCaseSequence[31].getNextStep({})).toBe('/respondent/task-list');
+    expect(respondentCaseSequence[31].getNextStep({})).toBe('/respondent/proceedings/summary');
```
Comment (yogendraU) [inline src/main/steps/respondent/respondentcaseSequence.test.ts:132]: Hi @ShakiraKP , Please do not change anything in respondent folder. 

We already have a folder in base branch for safety concerns. Please move all your code in that folder. 
https://github.com/hmcts/prl-citizen-frontend/tree/feature/PRL-1615-c100-rebuild-journey/src/main/steps/c100-rebuild/safety-concerns

Also follow the standards which we follow for in above folders. naming convention of folder, attributes

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/steps/respondent/respondentcaseSequence.test.ts (hunk 1 above)
Comment (ShakiraKP) [reply src/main/steps/respondent/respondentcaseSequence.test.ts]: Changes made @yogendraU

File: src/main/steps/c100-rebuild/safety-concerns/other-concerns/content.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+};
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    childSafetyConcerns: {
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/safety-concerns/other-concerns/content.ts:48]: @ShakiraKP Can we please follow the naming conventions defined for each features ?
Please prefix these fields with c1A_

File: src/main/steps/c100-rebuild/safety-concerns/other-concerns/content.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      classes: 'govuk-radios',
+      label: l => l.label,
+      section: l => l.section,
+      hint: l => l.hint,
+      values: [
+        {
+          label: l => l.one,
+          value: YesOrNo.YES,
+          subFields: {
+            childSafetyConcernsDetails: {
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/safety-concerns/other-concerns/content.ts:59]: Please prefix these fields with c1A_

File: src/main/app/case/case.ts [hunk 4]
```diff
@@ -200,6 +200,8 @@ export interface Case {
   //applicant1LanguagePreference?: LanguagePreference;
 
   safetyConcerns?: string;
+  childSafetyConcerns?: YesOrNo;
```
Comment (channa-patil) [inline src/main/app/case/case.ts:203]: @ShakiraKP We don't need to define fields unless we need to use them, fields are added userCase in session by default.
Can we please remove them if you are not using in sequence or any where else.

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/app/case/case.ts (hunk 4 above)
Comment (ShakiraKP) [reply src/main/app/case/case.ts:203]: Okay no problem, in adoption all values needed to be added to case data and added to cos api, I realise this is different here and I will no longer add these values to case data.

File: src/main/steps/c100-rebuild/safety-concerns/other-concerns/content.ts (hunk 3 above)
Comment (ShakiraKP) [reply src/main/steps/c100-rebuild/safety-concerns/other-concerns/content.ts:59]: alright np

**Review by channa-patil — APPROVED** [review]

**Pull request merged**